jack_bot.py

```diff
@@ -231,28 +172,6 @@ def set_game_step(self):
             # await self.prepare_building_positions(self.units(HATCHERY).first)
             self.prepare_expansions()
             self.split_workers()
-
-        if self.ground_enemies:
-            for hatch in self.townhalls:
-                close_enemy = self.ground_enemies.closer_than(25, hatch.position)
-                close_enemy_flying = self.known_enemy_units.flying.closer_than(30, hatch.position)
-                enemies = close_enemy.exclude_type({DRONE, SCV, PROBE})
-                enemies_flying = close_enemy_flying.exclude_type(self.excluded)
-                if enemies_flying and not self.counter_attack_vs_flying:
```

Comment:
thats missing too

Reply:
everything related with self.counter_attack_vs_flying is missing on this pr

Reply:
should be back in, please check